app/assets/javascripts/organization.coffee

```diff
@@ -0,0 +1,19 @@
+window.addAddressRow = (count) ->
+  newRow = $ """
+  <div class="form-group">
+    <label for="organization_addresses_attributes_""" + count + """_address">Mailing Address</label>
+
+    <div>
+      <input class="form-control" data-guard="different" type="text" value="" name="organization[addresses_attributes][""" + count + """][address]" id="organization_addresses_attributes_""" + count + """_address">
```

Comment:
Avoid the count in the name and instead use something like `name="organization[addresses][address][]"` which would map to something like `{ organization: { addresses: { address: ["123 main street", "555 broadway"] } } }`